app/cdap/components/PostRunActions/WizardNav/index.tsx

```diff
@@ -181,7 +181,7 @@ export const WizardNav = (props: IWizardNavProps) => {
           setPostActionsList(filterPlugins(res));
         },
         error: (err) => {
-          console.log('ERROR: ', err);
+          // console.log('ERROR: ', err);
```

Comment:
?

Reply:
if this isn't passing validation then add the exception comment for console log
```
    // tslint:disable-next-line: no-console
    ```

Reply:
gotcha, will add exceptions